services/property-svc/internal/property-value/projections/property_value_postgres_projection/property_value_postgres_projection.go

```diff
@@ -91,38 +86,28 @@ func (p *Projection) onPropertyValueCreated(ctx context.Context, evt hwes.Event)
 
 	repo := p.propertyValueRepo.WithTx(tx)
 
-	err, ack := createBasicPropertyValue(ctx, evt, repo, payload, propertyID, evt.AggregateID, subjectID, fieldType)
+	err, ack := createBasicPropertyValue(ctx, evt, repo, change, propertyID, evt.AggregateID, subjectID)
 	if err != nil {
 		return fmt.Errorf("onPropertyValueCreated: could not createbasicPropertyValue: %w", err), ack
 	}
 
-	if fieldType == pb.FieldType_FIELD_TYPE_SELECT || fieldType == pb.FieldType_FIELD_TYPE_MULTI_SELECT {
+	// for selects we are going to add options into the appropriate tables as well
+	isSingleSelectChange := change.SingleSelectValue != nil
+	isMultiSelectChange := change.MultiSelectValues != nil
+
+	if isSingleSelectChange || isMultiSelectChange {
 		var selectsToAdd []property_value_repo.ConnectValueWithSelectOptionsParams
 
-		if fieldType == pb.FieldType_FIELD_TYPE_SELECT {
-			val, ok := payload.Value.(string)
-			if !ok {
-				return errors.New("could not assert string"), hwutil.PtrTo(esdb.NackActionPark)
-			}
-			id, err := uuid.Parse(val)
+		if isSingleSelectChange {
+			id, err := uuid.Parse(*change.SingleSelectValue)
 			if err != nil {
-				return err, hwutil.PtrTo(esdb.NackActionPark)
+				return fmt.Errorf("option change is not a uuid: %w", err), hwutil.PtrTo(esdb.NackActionSkip)
```

Comment:
Why skip? We should park and investigate such issues.

Reply:
the payload will never be valid, so parking makes little sense